src/System.Private.ServiceModel/src/System/IdentityModel/Claims/ClaimSet.cs

```diff
@@ -31,6 +32,26 @@ public static ClaimSet System
             }
         }
 
+        public static ClaimSet Windows
+        {
+            get
+            {
+#if FEATURE_NETNATIVE // NegotiateStream
+                throw ExceptionHelper.PlatformNotSupported("Windows Stream Security is not supported on UWP yet");
```

Comment:
No complaints seen on my end when I built it in my TFS environment, but I'll double check that.